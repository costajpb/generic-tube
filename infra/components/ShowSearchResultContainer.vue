<template>
    <div :class="classes.container">
        <slot></slot>
    </div>
</template>

<style module="classes">
    .container {
        padding-bottom: var(--layout-margin-inline-default);

        @media (min-width: 700px) {
            padding: var(--layout-margin-block-compact);
            border: var(--border-size-3) solid var(--color-branding-secondary);
            border-radius: var(--radius-2);
            position: absolute;
            width: 100%;
            max-height: 60vh;
            overflow: auto;
            top: 100%;
            right: 0;
            /* FIXME */
            background: white;
            box-shadow: var(--shadow-6);
            border-radius: var(--radius-2);
        }
    }
</style>