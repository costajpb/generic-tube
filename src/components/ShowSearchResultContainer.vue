<template>
    <div :class="classes.container">
        <slot></slot>
    </div>
</template>

<style module="classes">
    .container {
        padding: var(--layout-margin-inline);
        box-sizing: border-box;

        @media (min-width: 700px) {
            padding: var(--layout-margin-inline);
            position: absolute;
            width: 100%;
            max-height: 60vh;
            overflow: auto;
            top: 100%;
            right: 0;
            /* FIXME */
            background: white;
            box-shadow: var(--shadow-6);
            border-radius: var(--radius-2);
        }
    }
</style>