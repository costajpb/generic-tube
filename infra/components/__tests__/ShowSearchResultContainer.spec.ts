import snapshowWrapper from '@/infra/__tests__/snapshotWrapper'
import ShowSearchResultContainer from '../ShowSearchResultContainer.vue'

describe('infra/components/ShowSearchResultContainer', () => {
  it('should render', () => {
    expect(snapshowWrapper(ShowSearchResultContainer)).toMatchSnapshot()
  })
})
