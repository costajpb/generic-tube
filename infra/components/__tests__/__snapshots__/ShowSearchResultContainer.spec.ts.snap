// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`infra/components/ShowSearchResultContainer > should render 1`] = `
<div
  class="_container_ea0614"
>
  
  
</div>
`;
